cypress/e2e/po/edit/settings.po.ts

```diff
@@ -36,6 +36,22 @@ export default class SettingsEditPo extends PagePo {
     selectSettings.clickOptionWithLabel(label);
   }
 
+  serverUrlLocalhostWarningBanner() {
+    return this.self().find('[data-testid="setting-serverurl-localhost-warning"]');
+  }
+
+  serverUrlTrailingForwardSlashErrorBanner() {
+    return this.self().find('[data-testid="Server URL should not have a trailing forward slash."]');
+  }
+
+  serverUrlNonHttpsErrorBanner() {
+    return this.self().find('[data-testid="Server URL must be https."]');
```

Comment:
@yonasberhe23 do you know if it's possible to add a conditional testid depending on the banner error, or assign a testid per banner? I'd prefer to use a locator rather find by text if possible.

Reply:
Yes, it looks like the locator is the label text in this case. The locator dynamically changes based on the error label. 

I do think these banner methods can be consolidated into one that accepts the label as an argument. Also, it should be using the `BannersPo`

e.g.
```
  bannerContent(label: string) {
    return new BannersPo('[data-testid="banner-content"]', this.self()).bannerElement(`${ label }`);
  }
```

Reply:
@yonasberhe23 yes, that's right.

I'll update the code to reflect your suggestions.